admin/fluent-bit/Makefile

```
@@ -0,0 +1,54 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=fluent-bit
+PKG_VERSION:=3.1.0
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/fluent/fluent-bit.git
+PKG_SOURCE_VERSION=v$(PKG_VERSION)
+PKG_MIRROR_HASH:=a9f7fbfae6fc8f8e5f738d868250daf7eafa24e8f652623cb9854bd0e16812ce
+
+PKG_LICENSE:=Apache-2.0
+PKG_LICENSE_FILES:=LICENSE
+
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/cmake.mk
+
+define Package/fluent-bit
+  SECTION:=admin
+  CATEGORY:=Administration
+  DEPENDS:= +libyaml +libopenssl +libcurl +libatomic +musl-fts +flex +bison
+  TITLE:=Fluent-Bit
+  URL:=https://fluentbit.io/
+endef
+
+define Package/fluent-bit/description
+  Fluent Bit is a super fast, lightweight, and highly scalable logging and metrics processor and forwarder.
```

```suggestion
  Fluent Bit is a super fast, lightweight, and highly scalable logging
  and metrics processor and forwarder.
```

wrap a new line for every ~70 chars

---

thanks for reviewing, done.